### ISSUE TITLE
Increase font size or line height

### ISSUE THREAD

**Issue by tomasberdych** [sitano/intellij-code-outline#11]
Please increase font-size or line height in code map. In the current version of the plugin is too small and poorly recognizable.

**Comment by MuppetGate** [comment]
Yeah, I'd like to see this too. At the moment, I'm just seeing large blobs of black.

**Comment by salain** [comment]
+1 for this

**Comment by rchrismon** [comment]
Looks like this is still open. If one has VS and installed Supercharger, there's a CodeMap, reproducing that functionality would be great!

In any event, the map needs to be large enough that I can see the method names. Then, I should be able to click on that item in the map and jump to that position.
